pom.xml

```diff
@@ -48,6 +51,39 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${maven-jacoco-plugin.version}</version>
+                <dependencies>
```

Comment:
Should we have a default rule of 0.75?  We do this in other repos

Reply:
I just put in what's currently there, we didn't add or modify any code in this PR.